Helper/Data.php

```diff
@@ -258,4 +258,13 @@ public function logError($exception)
             $this->log($exception);
         }
     }
+    
+    public function buildBackendParams($storeId) {
```

Comment:
Why are you building these here?

Reply:
backendParams require classes like ModuleListInterface (plugin version) and ProductMetadata (platform version) that are already loaded in the Data helper but the same Data helper is part of the Ajax controller constructor, so these params can be build in the controller instead of the helper. buildBackendParams function is removed.